src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,30 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_AGE = 18;
+    private static final int MINIMUM_PASS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() == null) {
+            throw new RuntimeException("Login can't be null");
```

Comment:
Task is not done. Fix your solution and re-request new review.

![image](https://user-images.githubusercontent.com/20326858/210063379-9c9086cd-a569-4afe-a8eb-cf60adce2343.png)